buku

```diff
@@ -3430,14 +3496,14 @@ def convert_bookmark_set(
             '<?xml version="1.0" encoding="UTF-8"?>\n'
             '<!DOCTYPE xbel PUBLIC \
 "+//IDN python.org//DTD XML Bookmark Exchange Language 1.0//EN//XML" \
-"http://pyxml.sourceforge.net/topics/dtds/xbel.dtd">\n'
+"http://pyxml.sourceforge.net/topics/dtds/xbel.dtd">\n\n'
             '<xbel version="1.0">\n')
 
         for row in resultset:
-            out += '<bookmark href="%s"' % (html.escape(row.url)).encode('ascii', 'xmlcharrefreplace').decode('utf-8')
+            out += '    <bookmark href="%s"' % (html.escape(row.url)).encode('ascii', 'xmlcharrefreplace').decode('utf-8')
             if row.tags:
                 out += ' TAGS="' + html.escape(row.tags).encode('ascii', 'xmlcharrefreplace').decode('utf-8') + '"'
-            out += '>\n<title>{}</title>\n</bookmark>\n'\
+            out += '>\n        <title>{}</title>\n    </bookmark>\n'\
```

Comment:
Using the same formatting as in the HTML export (…mostly for sake of appearances)